city_scrapers/spiders/chi_ssa_69.py

```diff
@@ -0,0 +1,130 @@
+import json
+import re
+from datetime import datetime, timedelta
+
+from city_scrapers_core.constants import COMMISSION, COMMITTEE, FORUM, NOT_CLASSIFIED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiSsa69Spider(CityScrapersSpider):
+    name = "chi_ssa_69"
+    agency = "Chicago Special Service Area #69 95th & Ashland Avenue"
+    timezone = "America/Chicago"
+
+    @property
+    def start_urls(self):
+        today = datetime.now()
+        last_week = today - timedelta(days=7)
+        in_two_months = today + timedelta(days=60)
+
+        # I think there is a problem where the timezone in Chicago will be
```

Comment:
Since it would only be an hour difference,and we're only looking for naive datetimes (in whatever the local timezone is), I think we can drop the timezone-aware datetimes here and just stick with a static offset. The worst case scenario is missing an event at the edge that is around midnight, which doesn't seem realistic

Reply:
Done.